custom.scss

```diff
@@ -1201,4 +1205,9 @@ html.res-commentBoxes .comment {
 
 .thing.stickied .md td, .thing.stickied .md th {
   border-color: #ababab
-}
\ No newline at end of file
+}
+
+.gilded {
+    box-shadow: 0 0 18px rgba(255,235,80,.4)!important;
```

Comment:
`!important` necessary here? This should be the last rule in the stylesheet, so more specificity may be needed if it's not applying.